apis/config/v1beta1/configuration_types.go

```diff
@@ -83,6 +83,10 @@ type Configuration struct {
 
 	// Resources provides additional configuration options for handling the resources.
 	Resources *Resources `json:"resources,omitempty"`
+
+	// featureGates is a map of feature names to bools that enable or disable
+	// alpha/experimental feature.
```

Comment:
```suggestion
	// featureGates is a map of feature names to bools that allows to override the
	// default enablement status of a feature. The map cannot be used in conjunction
	// with passing the list of features via the command line argument "--feature-gates"
	// for the Kueue Deployment.
```
I think alpha/experimental is unnecessary and confusing, the map can also be used to disable a beta feature.

Reply:
updated.